README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.org/mozilla-services/buildhub.svg?branch=master)](https://travis-ci.org/mozilla-services/buildhub)
 
-*Buildhub* aims to provide a public database of comprehensive information about releases and builds.
+_Buildhub_ aims to provide a public database of comprehensive information about releases and builds.
```

Comment:
Sorry. My editor uses Prettier on every save by default. Apparently this is the Prettier way of doing Markdown.